### PR TITLE
Add pre-defined event for the player discovering a rare

### PR DESCRIPTION
**Pull request by Schneebs**

### CODE REVIEW

File: PluginCore.Events.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ private void CharacterFilter_LoginComplete(object sender, EventArgs e)
                 Core.CharacterFilter.Death += CharacterFilter_Death;
 
                 TriggerWebhooksForEvent(EVENTS.LOGIN, Core.CharacterFilter.Name + " has logged in.");
+                ChatPatterns.Add(new ChatPattern(EVENTS.RARE, Core.CharacterFilter.Name + " has discovered "));
```
Comment (amoeba) [inline PluginCore.Events.cs]: The rest of the pre-defined chat patterns are defined over in

https://github.com/amoeba/TownCrier/blob/a279c21d982b5c57e87680988dc453890a4243ca/PluginCore.cs#L72-L76

is there a reason you wanted to add that here in the `LoginComplete` trigger? I think I'd rather have them in one place.

**Comment by amoeba** [comment]
Hey @Schneebs thanks for this. Looks good aside from the one comment about the location of where you added the chat trigger. Do you think you could move it over to `PluginCore.cs` with the others just to be consistent?

File: PluginCore.Events.cs (hunk 1 above)
Comment (Schneebs) [reply PluginCore.Events.cs]: Yeah, I tried putting it there first and beat my head against it for a bit.  I don't think the charfilter.name works until login is complete. When I tried it in PluginCore.cs it was catching anything matching " has discovered " and wasn't limiting captures to the name.

Comment (amoeba) [reply PluginCore.Events.cs]: You know, I think you're right! I'll merge this as-is.

**Comment by amoeba** [comment]
Thanks for adding this!

**Pull request merged**